dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -676,6 +683,9 @@ internal class CoreFeature(
     // endregion
 
     companion object {
+        internal const val SDK_INITIALIZED_IN_SECONDARY_PROCESS_WARNING_MESSAGE =
+            "Datadog SDK was initialized in a secondary process. We strongly recommend to initialize the SDK" +
+                "in the application main process. The sdk will not upload any data in this case."
```

Comment:
> [!NOTE]
> `We strongly recommend to initialize the SDK in the application main process`
> This part of the message makes it seem like initialising the SDK in a secondary process is wrong, when it's totally legitimate to want to capture events in a non main process. 
> Maybe we could phrase it differently, e.g.: 

```suggestion
            "Datadog SDK was initialized in a secondary process: although data will still be captured, nothing will be uploaded from this process." + 
            "Make sure to also initialize the SDK from the main process of your application."
```

Reply:
yes I agree this makes more sense. the first message was making more sense with my initial message where I explained the difference between running it in the main or main + secondary also.